pkg/didcomm/common/service/event.go

```diff
@@ -17,6 +17,15 @@ const (
 	PostState
 )
 
+// DIDMsg describes DIDCommMsg interface
+type DIDMsg interface {
```

Comment:
Assuming you move the implementation to a more appropriate package, would this be named `DIDCommMsg`?

Reply:
Sure!